src/object-list-to-trie.ts

```diff
@@ -0,0 +1,19 @@
+import { TrieOptions } from './types';
+import { createEmptyTrie, processCharSynonyms } from './trie';
+import { addObject } from './add-object';
+
+export function objectListToTrie<TValue extends object>(
+	list: Iterable<TValue>,
+	synonyms?: string[][],
+	reservedWords?: TrieOptions,
+) {
+	const parameters = processCharSynonyms(synonyms || [], reservedWords);
+	const trie = createEmptyTrie(parameters);
+	trie.s![3] = 1;
```

Comment:
What is the meaning of position '3'? A variable explaining the meaning before the assignment would be nice

Reply:
Position 1 indicates that the Trie is both accent and case insensitive.
You can create a Trie, though, and inconsistently add case-sensitive words to it, so this solution is not good yet.
I'm trying to figure out a better way to do it

Reply:
@DanielM08 I removed this options confusing array position and let it all specified on the TrieOptions (checkout the updated PR description).